diff --git a/src/Plugin/Linkit/Matcher/EntityMatcher.php b/src/Plugin/Linkit/Matcher/EntityMatcher.php
index 3383820..63c55ea 100644
--- a/src/Plugin/Linkit/Matcher/EntityMatcher.php
+++ b/src/Plugin/Linkit/Matcher/EntityMatcher.php
@@ -262,7 +262,7 @@ class EntityMatcher extends ConfigurableMatcherBase {
     $label_key = $entity_type->getKey('label');

     if ($label_key) {
-      $query->condition($label_key, '%' . $match . '%', 'LIKE');
+      $query->condition($label_key, $match, 'CONTAINS');
       $query->sort($label_key, 'ASC');
     }
