diff --git a/core/modules/views/src/Plugin/views/query/Sql.php b/core/modules/views/src/Plugin/views/query/Sql.php
index 03749a329b..f983f3f42d 100644
--- a/core/modules/views/src/Plugin/views/query/Sql.php
+++ b/core/modules/views/src/Plugin/views/query/Sql.php
@@ -1335,20 +1335,22 @@ public function query($get_count = FALSE) {
 
     // Make sure each entity table has the base field added so that the
     // entities can be loaded.
-    $entity_information = $this->getEntityTableInfo();
-    if ($entity_information) {
-      $params = [];
-      if ($groupby) {
-        // Handle grouping, by retrieving the minimum entity_id.
-        $params = [
-          'function' => 'min',
-        ];
-      }
+    if (!$this->view->getDisplay()->usesFields()) {
+      $entity_information = $this->getEntityTableInfo();
+      if ($entity_information) {
+        $params = [];
+        if ($groupby) {
+          // Handle grouping, by retrieving the minimum entity_id.
+          $params = [
+            'function' => 'min',
+          ];
+        }
 
-      foreach ($entity_information as $entity_type_id => $info) {
-        $entity_type = \Drupal::entityManager()->getDefinition($info['entity_type']);
-        $base_field = !$info['revision'] ? $entity_type->getKey('id') : $entity_type->getKey('revision');
-        $this->addField($info['alias'], $base_field, '', $params);
+        foreach ($entity_information as $entity_type_id => $info) {
+          $entity_type = \Drupal::entityManager()->getDefinition($info['entity_type']);
+          $base_field = !$info['revision'] ? $entity_type->getKey('id') : $entity_type->getKey('revision');
+          $this->addField($info['alias'], $base_field, '', $params);
+        }
       }
     }
 
