diff --git a/core/lib/Drupal/Core/Render/Element.php b/core/lib/Drupal/Core/Render/Element.php
index 75c3ab7401..f7e0e253db 100644
--- a/core/lib/Drupal/Core/Render/Element.php
+++ b/core/lib/Drupal/Core/Render/Element.php
@@ -94,7 +94,7 @@ public static function children(array &$elements, $sort = FALSE) {
         }
         // Only trigger an error if the value is not null.
         // @see https://www.drupal.org/node/1283892
-        elseif (isset($value)) {
+        elseif (isset($value) && (!empty($key) && $key != 0 && !is_int($key))) {
           trigger_error(new FormattableMarkup('"@key" is an invalid render array key', ['@key' => $key]), E_USER_ERROR);
         }
       }
