diff --git a/src/Controller/HandlePdfController.php b/src/Controller/HandlePdfController.php
index 44ffc59..682a3f4 100644
--- a/src/Controller/HandlePdfController.php
+++ b/src/Controller/HandlePdfController.php
@@ -264,6 +264,17 @@ class HandlePdfController extends ControllerBase {
         ->execute();
     }
 
+    // Protect data submitted by user in a webform.
+    // Delete the submission after the pdf is rendered.
+    if (isset($context['entity_ids']['webform_submission'])) {
+      $webform_submission_id = array_pop($context['entity_ids']['webform_submission']);
+      $webform_submission = \Drupal\webform\Entity\WebformSubmission::load($webform_submission_id);
+      $webform = $webform_submission->getWebform()->toArray();
+      if (isset($webform['category']) && $webform['category'] == 'fillpdf') {
+        $webform_submission->delete();
+      }
+    }
+
     return $response;
   }
 
